package.json

```diff
@@ -61,5 +61,8 @@
     "ts-jest": "^26.5.1",
     "tslib": "^2.0.3",
     "typescript": "^4.1.5"
+  },
+  "dependencies": {
+    "@testing-library/react": "^11.2.6"
```

Comment:
can you remove this as a dependency and add it as a devDependency?

`yarn remove @testing-library/react`
`yarn add -D @testing-library/react`